src/main/features/agreement/views/lot.njk

```diff
@@ -41,45 +27,38 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full govuk-grid-column-two-thirds-from-desktop">
         <div class="project-overview">
-            <p class="govuk-body">Agreement name: <strong>Digital Specialists and Programmes</strong></p>
-            <p class="govuk-body">Agreement ID: <strong>RM6263</strong></p>
-            <p class="govuk-body">Expiry date: <strong>DD month YYYY</strong></p>
+            <p class="govuk-body">Agreement name: <strong>{{agreement_lot.name}}</strong></p>
```

Comment:
This is not a lot name, it's agreement name

Reply:
You mean, it should be this:
![agreementNAME](https://user-images.githubusercontent.com/91210101/141655207-fba92248-18b9-46b6-8ea8-ff4e32d3f717.PNG)